pkg/chproducer/methods.go

```diff
@@ -319,7 +319,9 @@ func (h *Handler) fromBatchResponse(ctx context.Context, transferID string) (mod
 
 		for _, transaction := range blocks.Transactions {
 			if (transaction.FuncName == model.TxChannelTransferByCustomer.String() ||
-				transaction.FuncName == model.TxChannelTransferByAdmin.String()) &&
+				transaction.FuncName == model.TxChannelTransferByAdmin.String() ||
+				transaction.FuncName == model.TxChannelMultiTransferByCustomer.String() ||
+				transaction.FuncName == model.TxChannelMultiTransferByAdmin.String()) &&
```

Comment:
maybe this check and the analogous ones below should be put into a function?

Reply:
new package helpers/methods added. It contains some functions to check a kind of method.